scania-digital-design-system/sdds
# Improvement - add angular 12 and 13

**Describe pull-request**  
_Describe what the pull-request is about_
- Added angular 12 and 13 in angular demo folder, making easier for us to test
- Removed ignore .d.ts files for demo folder, since index.d.ts file is needed on newer versions of angular

**How to test**  
_Add description how to test if possible_
1. Go to angular folder
2. Run each version 11,12 and 13 make sure they start


**Additional context**  
_Add any other context about the pull-request here._
We should probably update them with all components
